tests/e2e-tests/README.md

```diff
@@ -7,8 +7,8 @@ Automated end-to-end tests for WooCommerce.
 - [Pre-requisites](#pre-requisites)
   - [Install NodeJS](#install-nodejs)
   - [Install Docker](#install-docker)
-  - [Configuration](#configuration)
-      - [Test Environment Configuration](#test-environment-configuration)
+  - [Configuration (how it works under the hood)](#configuration-(how it-works-under-the-hood))
```

Comment:
This change breaks the markdown!

Reply:
Thanks for catching it @ObliviousHarmony! I corrected it and added a few more details and explanations to the docs.